internal/promotion/runner/builtin/schemas/git-clone-config.json

```diff
@@ -14,6 +14,28 @@
       "description": "The URL of a remote Git repository to clone. Required.",
       "minLength": 1
     },
+    "user": {
+      "type": "object",
+      "description": "User information for the Git operations.",
+      "additionalProperties": false,
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the user performing the Git operations.",
+          "minLength": 1
+        },
+        "email": {
+          "type": "string",
+          "description": "The email of the user performing the Git operations.",
+          "format": "email"
+        },
+        "signingKey": {
+          "type": "string",
+          "description": "The GPG signing key for the user. Optional."
+        }
+      },
+      "required": ["name", "email"]
+    },
```

Comment:
To account for https://github.com/akuity/kargo/issues/3813#issuecomment-2828788694, please remember to add this to the schema for `git-commit` config as well.